crimemapper/tests/test_views.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding:utf-8 -*-
+"""Test file for views."""
+from crimemapper.views import map_view
+import pytest
+from webtest import AppError
+from pyramid.testing import DummyRequest
+
+
+def test_map_view_0(new_entry, clear_db_cache):
+    """Test that map view returns a list of items for template."""
+    result = map_view(DummyRequest)
+    assert type(result['places']) is list
+
+
+def test_map_view_1(new_entry, clear_db_cache):
```

Comment:
these numbered test functions are not particularly discoverable.  what do they test?  Can you name them so that their purpose is clear from their name?